docs/source/fetching/persisted-queries.mdx

```diff
@@ -0,0 +1,98 @@
+
+## Using Persisted Queries with Apollo iOS
+
+To enable persisted queries you will need to configure your code generation and `ApolloClient` correctly.
+
+### 1. Configure code generation for Persisted Queries
+
+To use persisted queries, your operations need to have a SHA256 hash identifier included in the generated operation objects. To configure code generation you will need to set the following 2 options in your `ApolloCodegenConfiguration` JSON file:
+
+In the `output` section of your configuration you will need to set the [`operationManifest`](./../code-generation/codegen-configuration#operation-manifest) with a version of either `persistedQueries` or `legacyAPQs`, and a path to output the manifest file.
+
+```json title="apollo-codegen-config.json"
+"output": {
+    "operationManifest" : {
+        "path" : "./generated/operationIdentifiers.json",
+        "version" : "persistedQueries"
+    }
+}
+```
+
+Then in the `options` section of your configuration you will need to set the `operationDocumentFormat` to be either `definition`, `operationId`, or both `definition` and `operationId`. This will determine how to generate the operation documents for your generated operations.
+
+```json title="apollo-codegen-config.json"
+"options": {
+    "operationDocumentFormat" : [
+        "definition",
+        "operationId"
+    ]
+}
+```
+
+The resulting operation manifest json would like this: 
+
+```json title="operationIdentifiers.json"
+{
+  "format": "apollo-persisted-query-manifest",
+  "version": 1,
+  "operations": [
+    {
+      "id": "e0321f6b438bb42c022f633d38c19549dea9a2d55c908f64c5c6cb8403442fef",
+      "body": "query GetItem { thing { __typename } }",
+      "name": "GetItem",
+      "type": "query"
+    }
+  ]
+}
+```
+
+> **Note:** This example is using the `persistedQueries` version of the operation manifest, the `legacyAPQs` version will look slightly different.
+
+### 2. Enable persisted queries on your `ApolloClient`
+
+You must also initialize your `ApolloClient` with a custom `NetworkTransport` that supports persisted queries.
+
+Initialize a `RequestChainNetworkTransport` with `autoPersistQueries` parameter set to `true` and an `interceptorProvider` that includes the `AutomaticPersistedQueryInterceptor` (such as `DefaultInterceptorProvider`).
+
+```swift
+let store = ApolloStore(cache: InMemoryNormalizedCache())
+let interceptorProvider = DefaultInterceptorProvider(store: store)
+let networkTransport = RequestChainNetworkTransport(
+    interceptorProvider: interceptorProvider,
+    endpointURL: URL(string: "http://localhost:4000/graphql")!
+)
+
+let client = ApolloClient(networkTransport: networkTransport, store: store)
+```
+For more information on configuring your `ApolloClient`, `NetworkTransport`, `InterceptorProvider`, and the request chain, see the [Request Pipeline](./../networking/request-pipeline) documentation.
+
+> **Note:** Sending APQs as `Get` requests
```

Comment:
Curious as to why this was made a note rather than [a separate section](https://www.apollographql.com/docs/ios/fetching/apqs/#send-apqs-as-get-requests) as in the current docs.